spec/lib/utils/job-utils/command-parser-spec.js

```diff
@@ -1147,5 +1147,85 @@ describe("Task Parser", function () {
             });
         });
     });
+
+    describe("driveId Parsers", function () {
+        it("should parse driveId output", function (done) {
+            var driveidCmd = 'sudo node get_driveid.js';
+
+            var tasks = [
+                {
+                    cmd: driveidCmd,
+                    stdout: stdoutMocks.driveidOutput,
+                    stderr: '',
+                    error: null
+                }
+            ];
+
+            taskParser.parseTasks(tasks)
+                .spread(function (result) {
+                    expect(result.error).to.be.undefined;
+                    expect(result.store).to.be.true;
+                    expect(result.source).to.equal('driveId');
+                    var driveIdLog = result.data;
+                    expect(driveIdLog).that.is.an('array').with.length(3);
+                    _.forEach(driveIdLog, function(elem) {
+                        expect(elem).contain.all.keys('identifier',
+                            'scsiId', 'virtualDisk', 'esxiWwid', 'linuxWwid', 'devName');
+                        expect(elem).property('identifier').to.be.a('number');
+                        expect(elem).property('devName').to.match(/sd[a-z]|hd[a-z]/);
```

Comment:
Since we have the sample file, we don't need to regex here, we can use the exact output value, not just output format.